### DISCUSSION TITLE
Purpose of long ID?

### DISCUSSION THREAD

**Discussion by petersem** [enchant97/hasty-paste#12] [Q&A] [answered]
What's the long ID option for? I understand the difference in the paste url, but wondered what the use case was?

**Comment by enchant97** [comment, marked as answer]
It is designed for reducing the possibility of a brute force attack.

**Comment by petersem** [comment]
Ah gotcha, I thought it may be associated with that. Thanks for the response. 🙂 Being that this is something that people would always set on or off, is this a better option to set as a container environment value, rather than take up UI real estate?

**Reply by enchant97** [reply]
It's definitely something that can be moved into a environment variable. Since it was the first release I just made it easy for people to toggle on and off without a restart.

**Reply by petersem** [reply]
Awesome, and thanks for a great app. Should I raise an issue for this enhancement?

**Reply by enchant97** [reply]
~Sure~, it's now implemented in commit: [d320394](https://github.com/enchant97/hasty-paste/commit/d3203941801d7c4f94168e0a14d838c31890ff8c)
